microsoftgraph/microsoft-graph-explorer-v4
# feat: migrate all components with status messaging

## Overview

Closes #3450 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

Best to test after the theming feature has been merged, else change theme to see status color changes.
